.github/workflows/chronos-prvn-python-spark-2.4-py37-onnxrt.yml

```diff
@@ -43,27 +43,14 @@ jobs:
       - name: Run Chronos python test
         shell: bash
         run: |
-          if conda info --env | grep "chronos-prvn-env-2"; then
-            source deactivate
-            conda remove -n chronos-prvn-env-2 -y --all
-          fi
+          conda remove -n chronos-prvn-env-2 -y --all
```

Comment:
what if chronos-prvn-env-2 does not exist, will **conda remove -n chronos-prvn-env-2 -y --all** fail?

Reply:
it seems the behavior of conda is just skip this cmd